checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -154,6 +154,13 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
                 org.checkerframework.checker.nullness.compatqual.MonotonicNonNullType.class,
                 MONOTONIC_NONNULL);
 
+        // Alias deprecated annotation
```

Comment:
I can't check for the keyfor version because that would introduce a dependency from the framework project to the checker project... unless I do a Class.forName, which would be rather ugly. Any objection to such an ugly hack?